### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by alasaidi** [HYF-Class22/agile-development-group2-hyf-redesign#1] [Ideas]
# Backlog

A collection of user stories you can choose from. Stories inside each priority
level are not necessarily in order, it's up to your group to decide how they fit
into your strategy. These are also just suggestion, feel free to change them or
create your own!

## Must-Haves

> these are necessary for basic usability

- [ ] As a user, I want to see the page's title
- [ ] As a user, I want to see the logo
- [ ] As a user, I want to see the header
- [ ] As a user, I want to see background 
- [ ] As a user, I want to see navigation bar for the website
- [ ] As a user, I want to see the footer
- [ ] As a user, I want to see main section with relevant information
  * who we are / what we do
  * curriculum + three programs 
  * achievement 
- [ ] As a user, I want to experience visually appealing and responsive website

## Should-Haves
- [ ] Semantic HTML 
- [ ] Accessibility features 

> these will complete the user experience, but are not necessary

As a user, I want to see the some links such as:
  - [ ] FAQ
  - [ ] contact
  - [ ] donation
  - [ ] success stories 
  - [ ] volunteer
  - [ ] social media 
 
## Could-Haves

> would be really cool ... if there's time

- [ ] animation
- [ ] video 
- [ ] blog page 
- [ ] information about coaches

**Comment by Prince4ademola** [comment]
Yeah we are good to go as we discussed,
